.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: c
+sudo: required
+dist: trusty
+compiler:
+  - gcc
+os: linux
+before_install:
+- sudo apt-get -qq update
+- sudo apt-get install -y dpkg-dev debhelper dkms lintian
+- sudo apt-get install linux-headers-`uname -r`
+- sudo cp -R . /usr/src/digimend-7
+script:
+- make
+- sudo dkms add -m digimend -v 7
+- sudo dkms build -m digimend -v 7
+- sudo dkms install -m digimend -v 7
+- sudo dkms uninstall -m digimend -v 7
+- dpkg-buildpackage -b -uc
+- lintian -i -I --profile debian --show-overrides ../digimend-dkms_*_amd64.changes
```

Comment:
Neat that you figured this out, but what does it do? Also, could you please make it not depend on the particular architecture we're building on?